gulp/tasks/clean.js

```diff
@@ -1,12 +1,20 @@
-const configClean = require( '../config' ).clean;
 const del = require( 'del' );
 const gulp = require( 'gulp' );
+const paths = require( '../../config/environment' ).paths;
 
 // Clean CSS out of /cfgov/static_built/css/
-gulp.task( 'clean:css', () => del( configClean.css + '/**/*' ) );
+gulp.task( 'clean:css', cb => {
+  del( paths.processed + '/css/**/*' );
+  del( paths.processed + '/apps/**/css/*' );
+  cb();
```

Comment:
You probably need `return cb()`. Does `gulp lint --quiet` complain about this?

Reply:
No complaints from my machine. Although now as I look at this, `del` is async so there's a race condition here. I'll refactor.